### ISSUE TITLE
Tetr.io Window QOL Improvements

### ISSUE THREAD

**Issue by bencecile** [tetrio/issues#814]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am certain this is a **feature request**, not a bug.
- [X] I've **thought this feature through** properly.

### Is your feature request related to a problem? Please describe.

Whenever I open Tetr.io on desktop, it seems to have a slightly random chance of changing the resolution from the last time it was opened. The location on the screen may also be different.

### Describe the solution you'd like

Keep track of the resolution and location of the window the last time it was closed (or you could put it with all the other settings depending).

### Describe alternatives you've considered

Being able to choose a resolution in the desktop related settings would also be good. If you decide to go this route, then it may be good to have a toggle to disable manual window scaling (or a "resolution" that would let you scale the window by hand, probably called "manual" or "automatic").

**Comment by ZaptorZap** [comment]
This definitely sounds like a bug report of unintended behavior. I can assure you, my Windows laptop remembers fullscreen preferences very well, and, this is sort of expected behavior from your OS in the first place. Are you plugging another monitor into your PC often, say for presentations perhaps?

**Comment by bencecile** [comment]
No, there are never other monitors. Also, this is for windowed mode only, not fullscreen. There's no resolution settings whatsoever, just fullscreen or not, which is why I thought of this ticket in the first place.

**Comment by HitGubber69** [comment]
I also reported this. For me the resolution keeps getting bigger and bigger every time tetr.io is launched, until eventually it takes up the whole screen

https://github.com/tetrio/issues/issues/747

**Comment by HitGubber69** [comment]
Still relevant
